test/unit/utils/api-request.spec.ts

```diff
@@ -92,6 +92,18 @@ function mockRequestWithError(err: any) {
     .replyWithError(err);
 }
 
+/**
+ * Returns a new RetryConfig instance for testing. This is same as the default
+ * RetryConfig, with the backOffFactor set to 0 to avoid delays.
+ *
+ * @return {RetryConfig} A new RetryConfig instance.
+ */
+function testRetryConfig(): RetryConfig {
```

Comment:
should this take a parameter whether to reset backOffFactor and then you can use it in all tests not testing default behavior but require backOffFactor?

Reply:
That's same as default config. So using `defaultRetryConfig()` directly.